release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3016,6 +3016,62 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, insufficient permissions led to the persistence of the `disk.csi.azure.com/agent-not-ready=value:NoExecute` taint after creating new {azure-first} disk Container Storage Interface (CSI) driver nodes. With this release, a fix disables the removal of the `not-ready` taint for {azure-short} disk CSI driver nodes, making the scheduler adhere to the `volume-attach-limit`. (link:https://issues.redhat.com/browse/OCPBUGS-54383[*OCPBUGS-54383*])
+
+* Previously, containers that used the SELinux domain of `container_logreader_t` to view container logs on a host at `/var/log` could not access logs in the `/var/log/containers` subdirectory. This was because of a missing symbolic link. With this release, a symbolic link is created for `/var/log/containers` so the containers can access the logs in `/var/log/containers`. (link:https://issues.redhat.com/browse/OCPBUGS-54342[*OCPBUGS-54342*])
+
+* Previously, an image pull secret that was generated for the internal Image Registry was not regenerated until after the embedded credentials expired This resulted in the image pull secrets being temporarily invalid. With this release, a fix for the image pull secrets are refreshed before the embedded credentials expire. (link:https://issues.redhat.com/browse/OCPBUGS-54304[*OCPBUGS-54304*])
+
+* Previously, the cluster autoscaler stopped scaling because of a failed machine in a machine set. This condition occurred due to inaccuracies in the way the cluster autoscaler counted machines in various non-running phases. With this release, the inaccuracies are fixed, allowing the cluster autoscaler to have an exact count. (link:https://issues.redhat.com/browse/OCPBUGS-53241[*OCPBUGS-53241*])
+
+* Previously, the Time-Group Management (T-GM) status was incorrectly announced as `S2` before the Digital Phase Locked Loop (DPLL) was locked during the Global Navigation Satellite System (GNSS) holdover. This caused inaccurate synchronization. With this release, the `DPLL` state decision logic is modified to ensure that the T-GM-STATUS moves to `S2` only after both phase offsets are valid and DPLL is in "Locked Holdover Acquired" state. This software update guarantees that the T-GM status accurately reflects the DPLL state when the GNSS source starts. (link:https://issues.redhat.com/browse/OCPBUGS-52956[*OCPBUGS-52956*])
+
+* Previously, a User-Defined Network (UDN) pod that is not qualified to use `egressIP` had to use its own UDN pod IP address instead of its node IP address as the source IP address in egressing packets. With this release, the UDN pod network is advertised correctly. (link:https://issues.redhat.com/browse/OCPBUGS-50965[*OCPBUGS-50965*])
```

Comment:
```suggestion
* Previously, a User-Defined Network (UDN) pod that is not qualified to use egress IP address had to use its own UDN pod IP address instead of its node IP address as the source IP address in egressing packets. With this release, the UDN pod network is advertised correctly. (link:https://issues.redhat.com/browse/OCPBUGS-50965[*OCPBUGS-50965*])
```